app/src/main/java/com/cornellappdev/android/eatery/WeeklyMenuFragment.java

```diff
@@ -0,0 +1,364 @@
+		// Inflate the layout for this fragment
+		View rootView = inflater.inflate(R.layout.fragment_weekly_menu, container, false);
+		presenter = new WeeklyPresenter(rootView);
+		breakfastText = rootView.findViewById(R.id.breakfast);
+		lunchText = rootView.findViewById(R.id.lunch);
+		dinnerText = rootView.findViewById(R.id.dinner);
+		linDate = rootView.findViewById(R.id.lin_date);
+		expListViewWest = rootView.findViewById(R.id.expandablelistview_west);
+		expListViewNorth = rootView.findViewById(R.id.expandablelistview_north);
+		expListViewCentral = rootView.findViewById(R.id.expandablelistview_central);
+		westText = rootView.findViewById(R.id.west_header);
+		northText = rootView.findViewById(R.id.north_header);
+		centralText = rootView.findViewById(R.id.central_header);
+
+		// Set OnClickListener for dates
+		date0 = rootView.findViewById(R.id.date0);
+		date1 = rootView.findViewById(R.id.date1);
+		date2 = rootView.findViewById(R.id.date2);
+		date3 = rootView.findViewById(R.id.date3);
+		date4 = rootView.findViewById(R.id.date4);
+		date5 = rootView.findViewById(R.id.date5);
+		date6 = rootView.findViewById(R.id.date6);
+
+		date0.setOnClickListener(this);
+		date1.setOnClickListener(this);
+		date2.setOnClickListener(this);
+		date3.setOnClickListener(this);
+		date4.setOnClickListener(this);
+		date5.setOnClickListener(this);
+		date6.setOnClickListener(this);
+
+		// Layout for menu list
+		getActivity().setTitle("Upcoming Menus");
+		DisplayMetrics dm = new DisplayMetrics();
+		getActivity().getWindowManager().getDefaultDisplay().getMetrics(dm);
+		int width = dm.widthPixels;
+		expListViewWest.setIndicatorBounds(width - 250, width);
+		expListViewNorth.setIndicatorBounds(width - 250, width);
+		expListViewCentral.setIndicatorBounds(width - 250, width);
+
+		lastExpandedPosition = -1;
+		lastClickedListView = null;
+
+		expListViewCentral.setOnGroupExpandListener(
+				new NonScrollExpandableListView.OnGroupExpandListener() {
+					@Override
+					public void onGroupExpand(int i) {
+						if (lastClickedListView != null
+								&& lastExpandedPosition != -1
+								&& i != lastExpandedPosition) {
+							lastClickedListView.collapseGroup(lastExpandedPosition);
+						}
+						lastExpandedPosition = i;
+						lastClickedListView = expListViewCentral;
+					}
+				});
+
+		expListViewWest.setOnGroupExpandListener(
+				new NonScrollExpandableListView.OnGroupExpandListener() {
+					@Override
+					public void onGroupExpand(int i) {
+						if (lastClickedListView != null
+								&& lastExpandedPosition != -1
+								&& i != lastExpandedPosition) {
+							lastClickedListView.collapseGroup(lastExpandedPosition);
+						}
+						lastExpandedPosition = i;
+						lastClickedListView = expListViewWest;
+					}
+				});
+		expListViewNorth.setOnGroupExpandListener(
+				new NonScrollExpandableListView.OnGroupExpandListener() {
+					@Override
+					public void onGroupExpand(int i) {
```

Comment:
in here and the two methods above, I'd like a more descriptive method parameter than `i`

Reply:
fixed!